src/main/java/Task.java

```diff
@@ -0,0 +1,126 @@
+public class Task {
+	protected String description;
+	protected boolean isDone;
+	public static final int MAX_STRING_SIZE = 100;
+	public static String[] lists = new String[MAX_STRING_SIZE];
+	private static int commandCount = 0;
+	private static int taskFinished = 0;
+	private static String taskDone;
+	protected String deadlineKeyword = "deadline";
+	protected String actionKeyword = "todo";
+	protected String eventKeyword = "event";
+
+	public Task(String description) {
+		this.description = description;
+		this.isDone = false;
+	}
+
+	public void storeCommand() {
+		if (isDeadline()) {
+			storeDeadline();
+		} else if (isAction()) {
+			storeTodo();
+		} else if (isEvent()) {
+			storeEvent();
+		}
+	}
+
+	public void echoCommand() {
+		echoCommandMain();
+		echoCommandEnding();
+	}
+
+	public void printList() {
```

Comment:
same as echoCommand, perhaps can consider commenting in the code to separate the 2 parts. since reader would have to go and look for the 2 methods, then be able to understand. decreased readability of code